utils/notify_community_pipelines_mirror.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+# Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+
+import requests
+
+
+# Configuration
+GITHUB_REPO = "huggingface/diffusers"
+GITHUB_RUN_ID = os.getenv("GITHUB_RUN_ID")
+SLACK_WEBHOOK_URL = os.getenv("SLACK_WEBHOOK_URL")
+
+
+def main(args):
+    action_url = f"https://github.com/{GITHUB_REPO}/actions/runs/{GITHUB_RUN_ID}"
+    if args.status == "success":
+        hub_path = "https://huggingface.co/datasets/diffusers/community-pipelines-mirror/tree/main"
```

Comment:
is there a way to report the actual path it mirrored to? 
for example the workflow https://github.com/huggingface/diffusers/actions/runs/9489094498/job/26149588467#step:8:19 is "successful" but wrong: you can see that 
CHECKOUT_REF: refs/tags/v0.29.0
PATH_IN_REPO:
 it pushed to `[https://huggingface.co/datasets/diffusers/community-pipelines-mirror/tree/main/.]`

compared with this one that's actually run correctly, the path is `https://huggingface.co/datasets/diffusers/community-pipelines-mirror/tree/main/v0.29.0` https://github.com/huggingface/diffusers/actions/runs/9495944132/job/26169275902#step:8:16